point/gemini_exceptions.py

```diff
@@ -0,0 +1,113 @@
+# Gemini2 Exception Inheritance Tree
+# ==================================
+#
+# Gemini2Exception
+#    G2BackendException
+#       G2BackendCommandNotSupportedError
+#       G2BackendFeatureNotSupportedError
+#       G2BackendFeatureNotImplementedYetError
+#       G2BackendCommandError
+#       G2BackendResponseError
+#       G2BackendReadTimeoutError
+#    G2CommandException
+#       G2CommandParameterError
+#          G2CommandParameterTypeError
+#    G2ResponseException
+#       G2ResponseDecodeError
+#          G2ResponseTooShortError
+#          G2ResponseMissingTerminatorError
+#          G2ResponseTooFewDelimitersError
+#          G2ResponseChecksumMismatchError
+#       G2ResponseParseError
+#          G2ResponseIntegerParseError
+#          G2ResponseAngleParseError
+#          G2ResponseTimeParseError
+#          G2ResponseRevisionsParseError
+#       G2ResponseBoundsViolation
+#          G2ResponseIntegerBoundsViolation
+#       G2ResponseInterpretationFailure
+#
```

Comment:
Since this is a brand-new file please use Google-style docstrings (see http://google.github.io/styleguide/pyguide.html) and make sure it passes pylint.